migrations/3_core.js

```diff
@@ -37,6 +38,9 @@ const KYBER_NETWORK_PROXY_ADDRESS_KOVAN = '0x7e6b8b9510d71bf8ef0f893902ebb9c865e
 const KYBER_NETWORK_PROXY_ADDRESS_ROPSTEN = '0x818e6fecd516ecc3849daf6845e3ec868087b755';
 const KYBER_NETOWRK_PROXY_ADDRESS_TESTRPC = '0x371b13d97f4bf77d724e78c16b7dc74099f40e84';
 
+const WETH_ADDRESS_KOVAN = '0x4C5E0CAbAA6B376D565cF2be865a03F43E361770';
+const WETH_ADDRESS_MAINNET = '';
```

Comment:
Should we start filling this in now?

Reply:
I'll add a card. There's more spots than just this to wrangle